configs/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for nextvibe project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.2/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'configs.settings.development')
```

Comment:
❗ `configs/settings/__init__.py` 파일에서 자동으로 설정되도록 해야 합니다. 이렇게 작성하면 development 설정으로만 실행됩니다.
```suggestion
os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'configs.settings')
```